oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/spi/editor/FulltextDocumentMaker.java

```diff
@@ -336,6 +338,32 @@ private boolean indexProperty(String path,
         return dirty;
     }
 
+    private String getValueAsTruncatedString(Object val) {
+        String value = val.toString();
+        return value.length() <= 128 ? value
+            : value.substring(0, 128) + " ... [" + value.hashCode() + "]";
```

Comment:
Do not add the three dots if the value is printed in full, add it only if the value is truncated.

Reply:
Same as the comment above - it won't be printed unless it exceed the truncation length.

Reply:
👍